Spanish translation.

phpbb-es opened the pull request:
Spanish translation.

phpbb-es committed:
Spanish translation.

Spanish translation.

nickvergessen commented on language/es/newspage.php:
```diff
@@ -0,0 +1,71 @@
+
+$lang = array_merge($lang, array(
+	'NEWS'						=> 'Noticias',
+	'NEWS_ADD_NEW'				=> 'New',
```
forgot to translate this?

nickvergessen commented:
Thanks for your translation, I'm yet undecided, how I will handle translations. But I guess I will find a good way for it.

phpbb-es committed:
Update newspage.php

From GitHub we can "control" translations. But if you think it's early to add translations, we can expect Joas.

No problem. ;)

phpbb-es commented:
Forgot to translate two lines...
